### DISCUSSION TITLE
Klick'r 3.1.0-beta02

### DISCUSSION THREAD

**Discussion by Nain57** [Nain57/Smart-AutoClicker#553] [Announcements]
> [!IMPORTANT]
> **This is a beta build.**
> It might be unstable so please backup your scenarios before updating to this version. It will be only availble on direct download here, and not on PlayStore/FDroid.

## Changes

* Fix Klick'r Overlay going invisible when Android quickly accept then remove the screen projection permission (follow up of https://github.com/Nain57/Smart-AutoClicker/issues/509)
* Fix long scenario name display in scenario screen
* Display fulfillment state instead of detection state in ImageCondition try screen (https://github.com/Nain57/Smart-AutoClicker/issues/551)

**Comment by MarioTn777** [comment]
I tried importing my project backup from the stable version of the app to the beta version, but it showed an error. My project is quite complex, so recreating it in the beta version would be very exhausting. Do you have any solutions? I need the notification feature available in the beta version.

**Reply by Nain57** [reply]
Hi and thank you for your feedback.

Can you provide the zip file of the problematic scenario ? This will help me find where the issue is .

**Reply by MarioTn777** [reply]
Thank you for your response. I have found the solution. I just needed to change the version in the previous backup zip file. I modified the version code inside the JSON file. Previously, it was version 13, and I changed it to version 14. After that, I was able to import the zip file into the beta version. Thanks!

**Reply by Nain57** [reply]
This is indeed the issue, import should be fixed in beta03, available as soon as the build finishes: https://github.com/Nain57/Smart-AutoClicker/releases/tag/3.1.0-beta03
